<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>阿巴阿巴</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="editAdd"/>
    <addaction name="editDel"/>
    <addaction name="editSave"/>
    <addaction name="editCancel"/>
    <addaction name="separator"/>
    <addaction name="editModity"/>
    <addaction name="editReadOnly"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="fileImp"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="fileAdd">
   <property name="icon">
    <iconset>
     <normaloff>:/images/images/new.png</normaloff>:/images/images/new.png</iconset>
   </property>
   <property name="text">
    <string>增加</string>
   </property>
   <property name="toolTip">
    <string>增加</string>
   </property>
  </action>
  <action name="editAdd">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>新增</string>
   </property>
  </action>
  <action name="editDel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="fileNew">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/images/new.png</normaloff>:/images/images/new.png</iconset>
   </property>
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="fileImp">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/images/images/new.png</normaloff>:/images/images/new.png</iconset>
   </property>
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="editModity">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
   <property name="toolTip">
    <string>修改</string>
   </property>
  </action>
  <action name="editReadOnly">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>只读</string>
   </property>
   <property name="toolTip">
    <string>只读</string>
   </property>
  </action>
  <action name="actDetailPage">
   <property name="icon">
    <iconset>
     <normaloff>:/images/images/new.png</normaloff>:/images/images/new.png</iconset>
   </property>
   <property name="text">
    <string>详情页</string>
   </property>
   <property name="toolTip">
    <string>打开详情页</string>
   </property>
  </action>
  <action name="editSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存</string>
   </property>
  </action>
  <action name="editCancel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="toolTip">
    <string>取消修改</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="toolTip">
    <string>关闭当前</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
